hadoop-mapreduce/src/test/java/org/apache/accumulo/hadoop/WithTestNames.java

```diff
@@ -16,7 +16,7 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.accumulo;
+package org.apache.accumulo.hadoop;
```

Comment:
This isn't unique enough to ensure that this package will only appear in the test classes/test jar. This package name is still one that could conceivable collide with a package in the main jar for this module if it is ever referenced by an integration test.

If it's only referenced by unit tests, that's fine, because jar sealing won't come into play... because the maven-surefire-plugin runs before the jars are built. However, integration tests run after the package phase when the jars are built, so the classes used for integration tests must not be in packages that could exist in the main jar.

Reply:
I'm not sure I fully understand but I think I moved them into a package that is only used for tests. The new package for this class reads `org.apache.accumulo.hadoop.its`